src/components/datepicker/js/datepickerDirective.js

```diff
@@ -89,13 +89,15 @@
             (ariaLabelValue ? 'aria-label="' + ariaLabelValue + '" ' : '') +
             'class="md-datepicker-input" ' +
             'aria-haspopup="true" ' +
+            'aria-expanded="{{!!ctrl.isCalendarOpen}}" ' +
```

Comment:
`isCalendarOpen` should be already a boolean, and also the expression automatically checks for truthiness.

> https://github.com/angular/material/blob/a73ab43c437ec7468d51da54f159b9f2845ec07b/src/components/datepicker/js/datepickerDirective.js#L298